testutils/include/cobs/testutils.h

```diff
@@ -0,0 +1,17 @@
+/* SPDX-License-Identifier: Apache-2.0 */
+
+#ifndef COBS_TESTUTILS_H
+#define COBS_TESTUTILS_H
+
+#include <cobs.h>
+
+void unsafe_cobs_reset_encode_pool(void);
+
+size_t cobs_encode_stream_simple(const uint8_t *const restrict input, const size_t input_length,
```

Comment:
(multiple) I should remove all of those restrict keywords. It's not clear if there is a real performance gain and I didn't think about if we're actually sure that pointers don't overlap.

Reply:
done